doc/uftrace-replay.md

```diff
@@ -242,6 +242,19 @@ Each field has following meaning:
 
 The default value is 'duration,tid'.  If given field name starts with "+", then it'll be appended to the default fields.  So "-f +time" is as same as "-f duration,tid,time".  And it also accepts a special field name of 'none' which disables the field display and shows function output only.
 
+    $ uftrace replay -f none
+    # FUNCTION
+     __monstartup();
+     __cxa_atexit();
+     main() {
+       a() {
+         b() {
+           c() {
+             getpid();
+           } /* c */
+         } /* b */
+       } /* a */
+     } /* main */
```

Comment:
I don't think this is needed.  It's not possible to have examples for every combination of options.  We already have an example for the -f option.